cmd/commands/runtime.go

```diff
@@ -380,71 +399,85 @@ func RunRuntimeInstall(ctx context.Context, opts *RuntimeInstallOptions) error {
+		return fmt.Errorf("failed to install components: %s\nfor more information: https://codefresh.io/csdp-docs/", err)
 	}
 
-	if err = RunGitSourceCreate(ctx, &GitSourceCreateOptions{
+	gitSrcMessage := fmt.Sprintf("Creating git source `%s`", store.Get().GitSourceName)
+	err = RunGitSourceCreate(ctx, &GitSourceCreateOptions{
 		InsCloneOpts:        opts.InsCloneOpts,
 		GsCloneOpts:         opts.GsCloneOpts,
 		GsName:              store.Get().GitSourceName,
 		RuntimeName:         opts.RuntimeName,
 		CreateDemoResources: opts.InstallDemoResources,
-	}); err != nil {
-		installationErr = fmt.Errorf("failed to create `%s`: %w", store.Get().GitSourceName, err)
-		return installationErr
+	})
+	appendLogToSummary(gitSrcMessage, err)
+	if err != nil {
+		return fmt.Errorf("failed to create `%s`: %w\nfor more information: https://codefresh.io/csdp-docs/", store.Get().GitSourceName, err)
 	}
 
 	mpCloneOpts := &git.CloneOptions{
 		Repo: store.Get().MarketplaceRepo,
 		FS:   fs.Create(memfs.New()),
 	}
 	mpCloneOpts.Parse()
-	if err = RunGitSourceCreate(ctx, &GitSourceCreateOptions{
+
+	createGitSrcMessgae := fmt.Sprintf("Creating %s", store.Get().MarketplaceGitSourceName)
+	err = RunGitSourceCreate(ctx, &GitSourceCreateOptions{
 		InsCloneOpts:        opts.InsCloneOpts,
 		GsCloneOpts:         mpCloneOpts,
 		GsName:              store.Get().MarketplaceGitSourceName,
 		RuntimeName:         opts.RuntimeName,
 		CreateDemoResources: false,
 		Exclude:             "**/images/**/*",
 		Include:             "workflows/**/*.yaml",
-	}); err != nil {
-		installationErr = fmt.Errorf("failed to create `%s`: %w", store.Get().MarketplaceGitSourceName, err)
-		return installationErr
+	})
+	appendLogToSummary(createGitSrcMessgae, err)
+	if err != nil {
+		return fmt.Errorf("failed to create `%s`: %w\nfor more information: https://codefresh.io/csdp-docs/", store.Get().MarketplaceGitSourceName, err)
 	}
 
-	if err = intervalCheckIsRuntimePersisted(ctx, opts.RuntimeName); err != nil {
-		installationErr = fmt.Errorf("failed to complete installation: %w", err)
-		return installationErr
+	timeoutErr := intervalCheckIsRuntimePersisted(ctx, opts.RuntimeName)
+	appendLogToSummary("Completing runtime installation", timeoutErr)
+	if timeoutErr != nil {
+		return fmt.Errorf("failed to complete installation: %w\nfor more information: https://codefresh.io/csdp-docs/", timeoutErr)
 	}
 
-	if err := addDefaultGitIntegration(ctx, opts.RuntimeName, opts.GitIntegrationOpts); err != nil {
-		return fmt.Errorf("failed to create default git integration: %w", err)
+	gitIntgErr := addDefaultGitIntegration(ctx, opts.RuntimeName, opts.GitIntegrationOpts)
+	appendLogToSummary("Creating a default git integration", gitIntgErr)
+	if gitIntgErr != nil {
+		return fmt.Errorf("failed to create default git integration: %w\nyou can create it manually with the command: `cf integration git add default --runtime %s --api-url https://github.com`", gitIntgErr, opts.RuntimeName)
```

Comment:
this is not the api url of github, and we can't assume the user is using github here anyway..
bottom line, I don't think we should print this message at all.